### PR TITLE
feat(7.7.5): add RN for 7.7.5 release

### PR DESCRIPTION
**Pull request by educhastenier**
+ describe with more details the performance improvements

### CODE REVIEW

File: md/release-notes.md [hunk 1]
```diff
@@ -84,7 +84,23 @@ Bonita binaries as well as Studio installer are now officially signed on Windows
...
 
-<a id="techonolgy-updates"/>
+### Performance (applies to Bonita 7.7.5+)
+The following improvements have been made to Bonita performance:
+- There is less I/O requests to temp folders
+- Work execution consumes less database resources
+- The login REST APIs mechanism consumes less resources
+- At engine startup and BPM services resume, the process dependencies loading consumes less memory
+
+A special attention has also been carried out to an improved deletion mechanism of archived cases. Deleting archived cases is now way more efficient.
```
Comment (NathalieC) [inline md/release-notes.md:94]: We should make it homogeneous with the chapter on 7.8.0.
Current proposition (waiting for review):
A few improvements have been brought to the overall Bonita efficiency:
- Deleting an archived case went down from seconds to milliseconds. Particularly enjoyable on environments with a large amount of archived data. Moreover, deleting cases in batches is now nearly as fast as deleting a single case and is not constrained by database connexion timeouts. 
- There is less I/O requests to temp folders
- Works execution uses less database resources
- The login REST APIs mechanism consumes less resources
- At engine startup and BPM services resume, the process dependencies loading requires less memory

File: md/release-notes.md [hunk 2]
```diff
@@ -84,7 +84,23 @@ Bonita binaries as well as Studio installer are now officially signed on Windows
...
+
+:::warning
```
Comment (NathalieC) [inline md/release-notes.md:96]: I thought we were about deprecating and deleting those event handlers. 
So there will be no long term option of reactivating it back.
The value (or lack of value) of these events should be pointed here so the reader realizes there is no point of wanting them.
I have not synchronized with CS before writing that. If you have, then your words must be kept.

Comment (NathalieC) [reply md/release-notes.md:96]: When we have an agreement on that, this must be added to the 7.8.0 release notes too.

**Comment by NathalieC** [comment]
More reviewers are needed to check my point of view.

File: md/release-notes.md (hunk 1 above)
Comment (educhastenier) [reply md/release-notes.md:94]: We plan to have the same content in 7.8, instead of the existing PR

File: md/release-notes.md (hunk 2 above)
Comment (educhastenier) [reply md/release-notes.md:96]: It will be reported to 7.8.0 RN, as soon as CS confirmes your remark above.

Comment (educhastenier) [reply md/release-notes.md:96]: Checked and integrated review from CS team...

**Pull request merged**